src/rust-crypto/rust-crypto.ts

```diff
@@ -1261,7 +1256,7 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
     public onSyncCompleted(syncState: OnSyncCompletedData): void {
         // Processing the /sync may have produced new outgoing requests which need sending, so kick off the outgoing
         // request loop, if it's not already running.
-        this.outgoingRequestLoop();
+        this.outgoingRequestsManager.doProcessOutgoingRequests();
```

Comment:
if you start an `async` method off without `await`ing, it, it's best to add a `.catch` to log any errors. (If for no other reason than it tells anyone reading the code that it is deliberately starting a background task.)

Reply:
added a .catch